x/mint/README.md

```diff
@@ -1,17 +1,16 @@
 # `x/mint`
 
-`x/mint` is a fork of the Cosmos SDK `x/mint` module that makes some changes to the inflation mechanism.
+`x/mint` is a fork of the Cosmos SDK `x/mint` module that makes some changes to the inflation mechanism. The changes were motivated by a desire for Celestia to have a pre-determined [inflation schedule](#inflation-schedule). See [ADR-019](../../docs/architecture/adr-019-strict-inflation-schedule.md) for more details.
```

Comment:
[Question] I thought we had decided to utilize a standardized template for the specifications of all the components in the app, similar to the one you created for the share specs. Has there been a change in the plan?

Reply:
[Suggestion] If we were to adopt a template similar to the one in the specs repository, it would be beneficial to incorporate a brief overview of the purpose of the inflation module, providing a high-level background, within the Abstract section.

Reply:
Discussed over Slack and let's adopt this template https://github.com/celestiaorg/celestia-app/blob/main/specs/template.md